google-http-client/src/main/java/com/google/api/client/http/HttpResponse.java

```diff
@@ -330,7 +336,8 @@ public InputStream getContent() throws IOException {
           String contentEncoding = this.contentEncoding;
```

Comment:
Existing i\ssue note: I don't see why we copy this into a local variable.

Reply:
Please see the latest commit.